Top-Level Requirements.md

```diff
@@ -77,10 +77,12 @@
 * 8.8 Upon completing the mining procedure, the mining subsystem **shall**:
```

Comment:
I think there is a slight omission here. Mining has the chance to be preempted. I think 8.8 should be reworded to:

* 8.8 Upon completing the mining procedure or user preemption, the mining subsystem **shall**: